frontend/docs/user-guide/configuration.md

```diff
@@ -0,0 +1,63 @@
+---
+outline: deep
+---
+
+# Configuration
+
+Users with Administrator permission can access {{ $params.APPLICATION_NAME }}’s Configuration page. This page is divided into three sections:
+
+1. Dynamic Configuration
+2. Dynamic Configuration History
+3. Deployment Configuration
+
+## Dynamic Configuration
+
+Admins can manage several features of {{ $params.APPLICATION_NAME }} without needing to make code changes or redeploy the application. The sections below will explore the different type of features that can be configured.
+
+Using the actions button on the right side of the page, an admin can:
+
+- **Import Configuration**: upload a JSON file with the correct schema and it will be deployed as the current dynamic configuration for {{ $params.APPLICATION_NAME }}.
+- **Export Configuration**: downloads a JSON file containing all of the current dynamic condifigurations for {{ $params.APPLICATION_NAME }}. This is useful if making a new deployment of {{ $params.APPLICATION_NAME }} and you wish to re-use the current configurations.
+- **Expand All**: expands all collapsed sections on the page so everything is in its expanded state.
+
+At the bottom of this page is a "Save Changes" button. Upon clicking this, the user will be shown a brief summary of the fields that have been changed and will be updated when this new dynamic configuration is uploaded.
+
+### Allowed Instance Types
+
+Modify the Amazon EC2 instance types that you wish to make available to users within the following resources: Notebook instance, Training and HPO jobs, Transform jobs, and Endpoints. IAM permissions that control access to these instance types within the {{ $params.APPLICATION_NAME }} user interface and Jupyter Notebooks will automatically update.
+
+### Activated Services
+
+Activate or deactivate services within {{ $params.APPLICATION_NAME }}. IAM permissions that control access to these services within the {{ $params.APPLICATION_NAME }} user interface and Jupyter Notebooks will automatically update. Deactivated services will no longer appear within the {{ $params.APPLICATION_NAME }} user interface. Deactivating services will terminate all active corresponding jobs and instances associated with the service.
```

Comment:
"terminate" -> "suspend"

Since this is the more detailed page, we may want to note that it will actually terminate EMR clusters because they are not able to be suspended

Reply:
Terminating a job is the same as stopping it, right? I think the wording is pretty accurate since disabling a service stops the jobs and terminates the EMR cluster (meaning the instances it uses).

Reply:
Yes and no. Really the only service where this is a truly strong distinction is notebooks. Notebooks can both be stopped and terminated. However, technically, this does just suspend the resources and is now the terminology we use on the admin page, so it is probably better to be consistent.